lib/babe/build.go

```diff
@@ -46,7 +51,17 @@ func (b *Service) buildBlock(parent *types.Header, slot Slot) (*types.Block, err
 		return nil, fmt.Errorf("failed to create block builder: %w", err)
 	}
 
+	startBuilt := time.Now()
+	defer func() {
```

Comment:
why is this in a defer?

Reply:
because I need to get the time after the `builder.buildBlock(...)` returns, but I've noticed that is necessary to check if `builder.buildBlock(...)` returns any error. I will remove from the defer and evaluate this metric after the `builder.buildBlock(...)` function call and create another metric for block builder errors.